File: Excel Sheet | Part - 1 - GFG/excel-sheet-part-1.java
//{ Driver Code Starts
//Initial Template for Java

/*package whatever //do not write package name here */

import java.io.*;
import java.util.*;

class GFG {
    public static void main (String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
	        
	    int t=Integer.parseInt(br.readLine());
    
	    while(t > 0){
	        int n = Integer.parseInt(br.readLine());
	        Solution T = new Solution();
	        
	        System.out.println(T.excelColumn(n));
	        
            t--;
	    }
	}
    
}
// } Driver Code Ends


//User function Template for Java

class Solution {
    public String excelColumn(int N){
        
        //  Your code here
        String res = "";
        while(N>0)
        {
            int d = N%26;
            if(d==0){
                res = 'Z'+ res;
                N = N/26 - 1;
            }
            else{
                res = (char)(64+d) + res;
                N = N/26;
            }
        }
        return res;
        
    }
}

File: Excel Sheet | Part - 1 - GFG/excel-sheet-part-1.cpp
//{ Driver Code Starts
//Initial template for C++


#include<bits/stdc++.h>
using namespace std;

// } Driver Code Ends
//User function template for C++

class Solution{
    public:
    string ExcelColumn(int N)
    {
        // Your code goes here
        string res = "";
        while(N)
        {
            int d = N%26;
            if(d==0){
                res = 'Z'+ res;
                N = N/26 - 1;
            }
            else{
                res = (char)(64+d) + res;
                N = N/26;
            }
        }
        return res;
    }
};

//{ Driver Code Starts.
int main()
{
    int t;
    cin>>t;
    while(t--)
    {
        int n;
        cin>>n;
        Solution ob;
        cout<<ob.ExcelColumn(n)<<endl;
    }
    return 0;
}
    
// } Driver Code Ends